.github/workflows/sycl-containers.yaml

```diff
@@ -69,6 +69,6 @@ jobs:
           password: ${{ secrets.GITHUB_TOKEN }}
           tags: |
             ghcr.io/${{ github.repository }}/${{ matrix.tags }}-${{ github.sha }}
```

Comment:
This line will probably have problems with tags on line 50 and 56.

Probably we should build a tag from both parameters file and tags.

ghcr.io/${{ github.repository }}/${{ matrix.file }}-${{ github.sha }}
ghcr.io/${{ github.repository }}/${{ matrix.file }}:${{ matrix.tags }}

and use `latest`, `devigc` and `unstable` as tags instead of `ubuntu2204_build`, `buntu2204_intel_drivers:devigc` and `ubuntu2204_intel_drivers:unstable`

Reply:
Agree. Done.